app/views/users/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= form_with(model: user, local: true) do |form| %>
+  <% if user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+      <% user.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
line 2-12 can be moved to another partial file like `_error_messages` and then just render that file for better readability

Reply:
I do not think it is important honestly, it is quite easy to understand that the code is displaying error messages.

Are you fine, if I leave it just like that?

Reply:
hmm ok then np hahaha